src/ImageIO/CGImageAnimation.cs

```diff
@@ -0,0 +1,141 @@
+//
+// ImageIO - CGImageAnimation.cs
+//
+// Authors:
+//	Whitney Schmidt  <whschm@microsoft.com>
+//
+// Copyright 2020, Microsoft Corp.
+//
+
+using System;
+using System.Runtime.InteropServices;
+using CoreFoundation;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace ImageIO
+{
+
+    public partial class CGImageAnimation
+    {
+
+        public delegate void CGImageSourceAnimationBlock (nint index, CGImage image, out bool stop);
+
+        [Introduced (PlatformName.MacOSX, 10, 15, PlatformArchitecture.All)]
+        [Introduced (PlatformName.iOS, 13, 0, PlatformArchitecture.All)]
+        [DllImport (Constants.ImageIOLibrary)]
+        static extern int CGAnimateImageAtURLWithBlock ( /* CFURLRef */ IntPtr url, /* CFDictionaryRef _iio_Nullable */ IntPtr options, /* CGImageSourceAnimationBlock */ ref BlockLiteral block);
+
+        [Introduced (PlatformName.MacOSX, 10, 15, PlatformArchitecture.All)]
+        [Introduced (PlatformName.iOS, 13, 0, PlatformArchitecture.All)]
+        [DllImport (Constants.ImageIOLibrary)]
+        static extern int CGAnimateImageDataWithBlock ( /* CFDataRef _Nonnull */ IntPtr data, /* CFDictionaryRef _Nullable */ IntPtr options, /* CGImageSourceAnimationBlock _Nonnull */ ref BlockLiteral block);
+
+        [Introduced (PlatformName.MacOSX, 10, 15, PlatformArchitecture.All)]
+        [Introduced (PlatformName.iOS, 13, 0, PlatformArchitecture.All)]
+        [BindingImpl (BindingImplOptions.Optimizable)]
+        public int AnimateImage (NSUrl url, NSDictionary options, [BlockProxy (typeof (NIDCGImageSourceAnimationBlock))] CGImageSourceAnimationBlock block)
```

Comment:
Same, let's avoid `Block` as a suffix or parameter name.

```suggestion
        public int AnimateImage (NSUrl url, NSDictionary options, [BlockProxy (typeof (NIDCGImageSourceAnimationBlock))] CGImageSourceAnimationHandler handler)
```